### PR TITLE
Documentation: Add warning to Neutrino backend config option

### PR DESCRIPTION
**Pull request by AbelLykens**
## Change Description
Related to  #6901 -- warn about backend change

## Steps to Test
N/A -- documentation

### Testing
- [ ] Your PR passes all CI checks.
- [ ] Tests covering the positive and negative (error paths) are included.
- [ ] Bug fixes contain tests triggering the bug to prevent regressions.

### Code Style and Documentation
- [ ] The change obeys the [Code Documentation and Commenting](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#CodeDocumentation) guidelines, and lines wrap at 80.
- [ ] Commits follow the [Ideal Git Commit Structure](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#IdealGitCommitStructure).
- [ ] Any new logging statements use an appropriate subsystem and logging level.
- [ ]  [There is a change description in the release notes](https://github.com/lightningnetwork/lnd/tree/master/docs/release-notes), or `[skip ci]` in the commit message for small changes.

📝 Please see our  [Contribution Guidelines](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md) for further guidance.

### CODE REVIEW

File: sample-lnd.conf [hunk 1]
```diff
@@ -489,10 +489,12 @@ bitcoin.simnet=true
 ; Use the btcd back-end
 bitcoin.node=btcd
 
-; Use the bitcoind back-end
+; Use the bitcoind back-end.
 ; bitcoin.node=bitcoind
 
 ; Use the neutrino (light client) back-end
+; WARNING: Please note switching from a full back-end (btcd/bitcoind) to
```
Comment (guggero) [inline sample-lnd.conf:496]: nit: add full stop in line above as well?
Also: please note _that_ switching...

**Review by guggero — COMMENTED** [review]
Thanks for the clarification. Just two small nits.

**Comment by AbelLykens** [comment]
s/note/note that
also fixed indentation + added newline, it now follows format of other `NOTE`s too.

**Review by guggero — COMMENTED** [review]
We should either remove the full stop (punctuation) on line 492 or add one to every sentence (at least within this block).

Also, commits should be squashed and prefixed according to the contribution guideline, thanks!

**Comment by AbelLykens** [comment]
Main things I do on git are `clone`, `pull`, `commit` and `push`. I think I put it in one commit now? Not sure though. I did add `docs:`. Also changed formatting a bit, the text at least to me makes sense now and I think this follows the formatting I see further on in the file.

**Comment by guggero** [comment]
> Main things I do on git are `clone`, `pull`, `commit` and `push`. I think I put it in one commit now? Not sure though. I did add `docs:`. Also changed formatting a bit, the text at least to me makes sense now and I think this follows the formatting I see further on in the file.

Hmm, I'm afraid those commands won't be enough in this case. What you'll need to do is an interactive rebase to squash the three commits into one. There's a handy guide in the Bitcoin Core docs: https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#squashing-commits

**Comment by AbelLykens** [comment]
I'll open a new PR, can't get this to work, did too many weird things.

**Comment by AbelLykens** [comment]
#6904 ;')

**Pull request closed without merging**